### ISSUE TITLE
Suggestion: wireless transmitter

### ISSUE THREAD

**Issue by penturt** [stfwi/redstonepen#56]
Not sure if it completely out of scope of the mod, but i would like to recommend some sort of wireless mechanic added to the mod. Nothing fancy just something to be able to trigger redstone remotely.

**Comment by stfwi** [comment]
Hi, it may be included in form of the Levers/Buttons of the mod, but I am still not sure if that is OP. Had wireless in the Gauges & Switches mod, and I am still not sure/hesitant to add this to a vanillarish mod like Redstone Pen. From a technical perspective it's much less hassle than actual Redstone wiring. Up-vote taken ;)

**Comment by penturt** [comment]
yeah i totally get it, i used to pair redstone pen with Synchronous Switches, and i think it kept it vailla-ish... but that mod hasn't updated and it kind of looks like it wont be any time soon

**Comment by stfwi** [comment]
As a first element to slowly approach the CR, there is now a manual "Redstone Remote" in the mod, that allows you to flip levers and push buttons remotely. Also Redstone Logic Controls can be toggled on and off with that. This feature is mainly intended for testing contraptions or automation setups, but transferring the functionality to special buttons and levers is not far fetched. The "is it OP for this vanillarish mod" is still in discussion though. Cheers;;

**Comment by penturt** [comment]
Yeah the OP-ness of these things are always questionable. Even a simple thing can tip it over that line.

**Comment by stfwi** [comment]
Closing this issue due to mod EOL.

**Comment by penturt** [comment]
Are you no longer going to support the mod?

**Comment by stfwi** [comment]
I am afraid so ; Time constrains and braking changes spoil the coding fun ;). I am sure the idea will be picked up by someone. Cheers,-

**Comment by penturt** [comment]
Completely understand, best on your endeavors.
